include/skystring.h

```diff
@@ -0,0 +1,15 @@
+#ifndef LIBSKY_STRING_H
+#define LIBSKY_STRING_H
+
+#include <stdio.h>
+#include <stdlib.h>
+#include "libskycoin.h"
+
+extern void randBytes(GoSlice *bytes, size_t n);
```

Comment:
Is this method only for tests? 

It its public to everyone it should use the cipher rand methods and not math/rand

Reply:
Only for testing purposes . Maybe we should have two header include paths : the one for release and a another just for the test suite ... @gz-c makes sense ?